src/administration/integrations/gluescript.md

```diff
@@ -0,0 +1,249 @@
+# GlueScript
+
+GlueScript scripts are powered by a lightweight Javascript engine that allows you to run a script on any activity within Platform.sh. This opens a whole world of new automation use-cases including, but not limited to, calling external web services.
+
+This is not a full-blown Javascript runtime, and you can't use NPM or import libraries. Keep it short and sweet.
+
+This is very much an **Alpha release**, and while the Javascript API itself will probably be stable enough, the script management will undoubtedly change. For the moment you would simply post a `.js` file to our REST API to create the integration. 
+
+> **CAVEAT**: Currently errors are silent so this is not for the faint of heart.
+
+While you don't get to have NPM GlueScript comes with a bunch of helpers that make it very useful. You can currently access the following APIs: [Project](#project-api), [Activity](#activity-api), [Process](#process-api), [Storage](#storage-api), [Fetch](#crypto-api) and [Crypto](#crypto-api)
+
+# Setting up a GlueScript integration
+
+## 1. Write the script
+
+Create a new `scripts` folder within the `.platform/` folder.
+
+```bash
+mkdir .platform/scripts
+```
+
+Create a new Javascript file inside the `scripts` folder with the following content:
+
+```javascript
```

Comment:
You don't need this, because the platform cli takes care of embedding the js file later on